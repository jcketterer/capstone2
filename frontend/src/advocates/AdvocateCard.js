import React from 'react';
import { Link } from 'react-router-dom';

const AdvocateCard = ({ advocate }) => {
  return (
    <div className="card">
      <div className="card-body">
        <Link to={`/advocate/${advocate.advocateId}`}>
          <div className="card-title">
            {advocate.firstName} {advocate.lastName}
          </div>
        </Link>
      </div>
    </div>
  );
};

export default AdvocateCard;
