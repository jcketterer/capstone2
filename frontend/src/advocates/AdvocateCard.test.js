import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import AdvocateCard from './AdvocateCard';

it('should render without breaking', () => {
  render(
    <MemoryRouter>
      <AdvocateCard advocate={{}} />
    </MemoryRouter>
  );
});

it('should render and match snapshot', () => {
  const { asFragment } = render(
    <MemoryRouter>
      <AdvocateCard advocate={{}} />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
