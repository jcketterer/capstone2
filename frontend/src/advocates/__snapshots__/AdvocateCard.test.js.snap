// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render and match snapshot 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <div
      class="card-body"
    >
      <a
        href="/advocate/undefined"
      >
        <div
          class="card-title"
        />
      </a>
    </div>
  </div>
</DocumentFragment>
`;
